src/transformers/models/longt5/modeling_longt5.py

```diff
@@ -1199,6 +1199,11 @@ def forward(
         hidden_states, present_key_value_state = self_attention_outputs[:2]
         attention_outputs = self_attention_outputs[2:]  # Keep self-attention outputs and relative position weights
 
+        # clamp inf values to enable fp16 training
```

Comment:
```suggestion
        # clamp inf values to enable fp16 inference - check https://github.com/huggingface/transformers/pull/19229/
```
Maybe let's specify that it's just for inference?